docker-compose/bind9/config/yourdomain.zone

```diff
@@ -0,0 +1,15 @@
+$TTL   300  ; 5 minutes
+$ORIGIN yourdomain.home.
+@       IN      SOA     yourdomain.home. (
+                       2024021700      ; serial
+                       12h             ; refresh
+                       15m             ; retry
+                       3w              ;expire
+                       2h              ;minimum ttl
```

Comment:
The meaning of the 5th field of a SOA record isn't "mimimum TTL" anymore, but it's rather the negative response cachine TTL, see RFC 2308.